zetaclient/chains/zrepo/zrepo.go

```diff
@@ -37,16 +88,257 @@ func (repo *ZetaRepo) GetOutboundTrackers(ctx context.Context) ([]types.Outbound
 	return trackers, nil
 }
 
+// TODO: We should probably move this to the TSS repository.
+func (repo *ZetaRepo) GetBTCTSSAddress(ctx context.Context) (string, error) {
+	address, err := repo.client.GetBTCTSSAddress(ctx, repo.connectedChain.ChainId)
+	if err != nil {
+		chainID := repo.connectedChain.ChainId
+		outterErr := fmt.Errorf("%w for chain %d", ErrClientGetBTCTSSAddress, chainID)
+		return "", newClientError(outterErr, err)
+	}
+	return address, nil
+}
+
+// ------------------------------------------------------------------------------------------------
+// Voting & Posting Trackers
+// ------------------------------------------------------------------------------------------------
+
+// PostOutboundTracker posts an outbound tracker.
+// It returns the hash of the associated ZetaChain transaction.
+func (repo *ZetaRepo) PostOutboundTracker(ctx context.Context, logger zerolog.Logger,
+	nonce uint64,
+	txHash string,
+) (string, error) {
+	// Does not post outbound trackers in dry mode.
+	if repo.clientMode == mode.DryMode {
+		logger.Info().Stringer(logs.FieldMode, mode.DryMode).Msg("skipping outbound tracker")
+		return "", nil
+	}
```

Comment:
We should document the differences between the different modes somwhere , preferably in the docs directory